pennylane/devices/__init__.py

```diff
@@ -163,5 +164,270 @@ def execute(self, circuits, execution_config = qml.devices.DefaultExecutionConfi
 from .default_clifford import DefaultClifford
 from .default_tensor import DefaultTensor
 from .null_qubit import NullQubit
+from .default_qutrit import DefaultQutrit
 from .default_qutrit_mixed import DefaultQutritMixed
 from .._device import Device as LegacyDevice
+from .._device import DeviceError
+
+
+def _get_device_entrypoints():
```

Comment:
Potentially we could add a `pennylane/devices/device_constructor.py` file and add this code there.

Reply:
Done 👌